data/cli.py

```diff
@@ -12,9 +12,20 @@
 from data import models
 from data.preprocess import pull_data, insert_data
 
+from azure.keyvault import KeyVaultClient
+from msrestazure.azure_active_directory import MSIAuthentication, ServicePrincipalCredentials
 
 LOGGER = logger.get_logger(__name__)
 
+# init Azure MSI & KeyVault creds
+
+if os.environ.get("TRACKER_KEYVAULT_URI", None) != None:
+    creds = MSIAuthentication(resource='https://vault.azure.net')
+    keyvault = KeyVaultClient(creds)
```

Comment:
```suggestion
    keyvault = KeyVaultClient(creds) # pylint: disable=invalid-name
```

These seen minimal enough to not bend over backwards to "fix" the linting error, I suggest just telling pylint to ignore the `invalid-name` errors.

The other ones I think should be fixed though

Reply:
On the one hand, it is a bit obnoxious to fail CI for formatting, but on the other hand if it's not done I almost guarantee that formatting issues will creep in over time.

It was quite a lot of work to get clear up all the past formatting issues, it would be a shame to let them back in.